behandlingsprosess/src/main/java/no/nav/ung/sak/domene/behandling/steg/varselrevurdering/VarselRevurderingAksjonspunktUtleder.java

```diff
@@ -0,0 +1,84 @@
+package no.nav.ung.sak.domene.behandling.steg.varselrevurdering;
+
+import no.nav.ung.kodeverk.behandling.BehandlingÅrsakType;
+import no.nav.ung.kodeverk.behandling.aksjonspunkt.Venteårsak;
+import no.nav.ung.kodeverk.ungdomsytelse.periodeendring.UngdomsytelsePeriodeEndringType;
+import no.nav.ung.sak.behandlingskontroll.AksjonspunktResultat;
+import no.nav.ung.sak.behandlingslager.behandling.aksjonspunkt.Aksjonspunkt;
+import no.nav.ung.sak.behandlingslager.behandling.motattdokument.MottattDokument;
+import no.nav.ung.sak.behandlingslager.behandling.startdato.UngdomsytelseBekreftetPeriodeEndring;
+import no.nav.ung.sak.domene.typer.tid.DatoIntervallEntitet;
+
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static no.nav.ung.kodeverk.behandling.BehandlingÅrsakType.RE_HENDELSE_ENDRET_STARTDATO_UNGDOMSPROGRAM;
+import static no.nav.ung.kodeverk.behandling.BehandlingÅrsakType.RE_HENDELSE_OPPHØR_UNGDOMSPROGRAM;
+import static no.nav.ung.kodeverk.behandling.aksjonspunkt.AksjonspunktDefinisjon.AUTO_SATT_PÅ_VENT_REVURDERING;
+
+public class VarselRevurderingAksjonspunktUtleder {
+
+    public static Optional<AksjonspunktResultat> utledAksjonspunkt(
+        List<BehandlingÅrsakType> behandlingsårsakerForBehandling,
+        List<DatoIntervallEntitet> perioder,
```

Comment:
Enda bedre å ta in en LocalDateTimeline<> her

Reply:
Det spørst vel om vi vil kjøre compress på det eller ikke først kanskje